### PR TITLE
[zh-cn]: update the translation of `Date.setUTCHours()` method

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setUTCHours
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/date/setutchours/index.md
* Last commit: https://github.com/mdn/content/commit/fb85334ffa4a2c88d209b1074909bee0e0abd57a

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md [hunk 1]
```diff
@@ -1,47 +1,55 @@
...
 title: Date.prototype.setUTCHours()
 slug: Web/JavaScript/Reference/Global_Objects/Date/setUTCHours
+l10n:
+  sourceCommit: fb85334ffa4a2c88d209b1074909bee0e0abd57a
 ---
 
 {{JSRef}}
 
-The **`setUTCHours()`** method sets the hour for a specified date according to universal time, and returns the number of milliseconds since 1 January 1970 00:00:00 UTC until the time represented by the updated {{jsxref("Date")}} instance.
+**`setUTCHours()`** 方法是 {{jsxref("Date")}} 实例的方法，它根据世界标准时间更改该日期的小时、分钟、秒或毫秒。
```
Comment (T34-active) [inline files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md:10]:
```suggestion
{{jsxref("Date")}} 实例的 **`setUTCHours()`** 方法根据世界协调时（UTC）更改该日期的小时、分钟、秒或毫秒。
```

File: files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md [hunk 2]
```diff
@@ -1,47 +1,55 @@
...
 title: Date.prototype.setUTCHours()
 slug: Web/JavaScript/Reference/Global_Objects/Date/setUTCHours
+l10n:
+  sourceCommit: fb85334ffa4a2c88d209b1074909bee0e0abd57a
 ---
 
 {{JSRef}}
 
-The **`setUTCHours()`** method sets the hour for a specified date according to universal time, and returns the number of milliseconds since 1 January 1970 00:00:00 UTC until the time represented by the updated {{jsxref("Date")}} instance.
+{{jsxref("Date")}} 实例的 **`setUTCHours()`** 方法根据世界协调时（UTC）更改该日期的小时、分钟、秒或毫秒。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md:10]: 未同步翻译

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md (hunk 2 above)
Comment (T34-active) [inline files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md:10]:
```suggestion
{{jsxref("Date")}} 方法是 **`setUTCHours()`** 实例的方法，用于根据世界协调时（UTC）更改日期对象的小时、分钟、秒或毫秒。
```

File: files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md [hunk 3]
```diff
@@ -1,47 +1,55 @@
...
 title: Date.prototype.setUTCHours()
 slug: Web/JavaScript/Reference/Global_Objects/Date/setUTCHours
+l10n:
+  sourceCommit: fb85334ffa4a2c88d209b1074909bee0e0abd57a
 ---
 
 {{JSRef}}
 
-The **`setUTCHours()`** method sets the hour for a specified date according to universal time, and returns the number of milliseconds since 1 January 1970 00:00:00 UTC until the time represented by the updated {{jsxref("Date")}} instance.
+{{jsxref("Date")}} 方法是 **`setUTCHours()`** 实例的方法，用于根据世界协调时（UTC）更改日期对象的小时、分钟、秒或毫秒。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md:10]:
```suggestion
{{jsxref("Date")}} 实例的 **`setUTCHours()`** 方法用于根据世界协调时更改日期对象的小时、分钟、秒或毫秒。
```

File: files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md [hunk 4]
```diff
@@ -1,47 +1,55 @@
...
-- `msValue`
-  - : 可选参数。表示毫秒的整数，取值 0 到 999 之间。如果指定了该参数，就要指定 `minutesValue` 和 `secondsValue` 这两个参数。
+  - : 0 到 23 之间的整数，表示小时数。
+- `minutesValue` {{optional_inline}}
+  - : 0 到 59 之间的整数，表示分钟数。
+- `secondsValue` {{optional_inline}}
+  - : 0 到 59 之间的整数，代表秒数。如果指定了 `secondsValue`，则必须同时指定 `minutesValue`。
+- `msValue` {{optional_inline}}
+  - : 0 到 999 之间的整数，表示毫秒数。
+  - 如果指定了 `msValue`，则必须同时指定 `minutesValue` 和 `secondsValue`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md:33]:
```suggestion
  - : 0 到 999 之间的整数，表示毫秒数。如果指定了 `msValue`，则必须同时指定 `minutesValue` 和 `secondsValue`。
```

File: files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md [hunk 5]
```diff
@@ -1,47 +1,55 @@
...
 
 ### 返回值
 
-返回从 1970-01-01 00:00:00 UTC 到更新后的日期所表示时间的毫秒数。
+该方法会直接修改 {{jsxref("Date")}} 对象，并返回其新的[时间戳](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Date#时代_时间戳和无效日期)。
+
+如果参数为 `NaN`（或其他会被[强制转换](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)为 `NaN`，例如 `undefined`），则日期会被设置为[无效日期](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Date#时代_时间戳和无效日期)，并返回 `NaN`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/date/setutchours/index.md:39]:
```suggestion
该方法会原地修改 {{jsxref("Date")}} 对象，并返回其新的[时间戳](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Date#纪元、时间戳和无效日期)。如果参数为 `NaN`（或其他会被[强制转换](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)为 `NaN` 的值，例如 `undefined`），则日期会被设置为[无效日期](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Date#纪元、时间戳和无效日期)，并返回 `NaN`。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
劳烦对一下，特别是我明确指出后修改的第一句的翻译，还是有点错误

**Review by yin1999 — APPROVED** [review]

**Pull request merged**